lib/manager/helmfile/extract.ts

```diff
@@ -0,0 +1,77 @@
+import is from '@sindresorhus/is';
+import yaml from 'js-yaml';
+
+import { logger } from '../../logger';
+import { PackageFile, PackageDependency, ExtractConfig } from '../common';
+
+export function extractPackageFile(
+  content: string,
+  fileName: string,
+  config: ExtractConfig
+): PackageFile {
+  let deps = [];
+  let doc;
+  const aliases: Record<string, string> = {};
+  try {
+    doc = yaml.safeLoad(content, { json: true });
+  } catch (err) {
+    logger.debug({ fileName }, 'Failed to parse helmfile helmfile.yaml');
+    return null;
+  }
+  if (!(doc && is.array(doc.releases))) {
+    logger.debug({ fileName }, 'helmfile.yaml has no releases');
+    return null;
+  }
+
+  if (doc.repositories) {
+    for (let i = 0; i < doc.repositories.length; i += 1) {
+      aliases[doc.repositories[i].name] = doc.repositories[i].url;
+    }
+  }
+  logger.debug({ aliases }, 'repositories discovered.');
+
+  deps = doc.releases.map(dep => {
+    let depName = dep.chart;
+    let repoName = null;
+
+    if (dep.chart.startsWith('./')) {
+      return {
+        depName,
+        skipReason: 'local-chart',
+      } as PackageDependency;
+    }
+
+    if (dep.chart.includes('/')) {
+      const v = dep.chart.split('/');
+      depName = v[1];
+      repoName = v[0];
+    } else {
+      repoName = dep.chart;
+    }
+
+    const res: PackageDependency = {
+      depName,
+      currentValue: dep.version,
+      registryUrls: [aliases[repoName]]
+        .concat([config.aliases[repoName]])
+        .filter((v, _) => {
```

Comment:
```suggestion
        .filter(Boolean);
```

I think the above works and is cleaner than the current 3-line approach.